packages/honeycomb-layout/package.json

```diff
@@ -10,19 +10,16 @@
     "start": "npm run start:production",
     "start:development": "cross-env NODE_ENV=development nodemon src/index.js --exec babel-node --watch src/ --ext js",
     "start:production": "cross-env NODE_ENV=production node dist/index.js",
-    "test": "npm run lint && npm run test:unit",
```

Comment:
I added a test for `honeycomb-layout` ➡️ 2f037012e0363421cb9c5970b4ff14f85250694e